python/tk_alias/scene_data_validator.py

```diff
@@ -172,7 +172,7 @@ def get_validation_data(self):
             "shader_unused": {
                 "name": "Delete Unused Shaders",
                 "description": "Check for shaders that are not assigned to any geometry.<br>"
-                               "The DefaultShader will be skipped.",
+                "The DefaultShader will be skipped.",
```

Comment:
I think if you want to have it formatted as before to make it more readable, you can use triple quotes`"""`

Reply:
Thanks @staceyoue , that works so I updated them all now. :)